app/javascript/flavours/glitch/initial_state.js

```diff
@@ -120,8 +120,10 @@ export const expandSpoilers = getMeta('expand_spoilers');
 export const forceSingleColumn = !getMeta('advanced_layout');
 export const limitedFederationMode = getMeta('limited_federation_mode');
 export const mascot = getMeta('mascot');
+export const maxReactions = (initialState && initialState.max_reactions) || 1;
 export const me = getMeta('me');
 export const movedToAccountId = getMeta('moved_to_account_id');
+export const visibleReactions = getMeta('visible_reactions');
```

Comment:
Good idea, I didn't notice that because RubyMine unfortunately doesn't support ESLint.  On that note, it might be a good idea to upgrade to TypeScript.  Not really relevant here, but something to maybe think about in the future.